test/stablex/batch_exchange.js

```diff
@@ -1515,6 +1515,106 @@ contract("BatchExchange", async accounts => {
       )
       assert.equal(0, (await batchExchange.currentPrices.call(2)).toString(), "CurrentPrice were not adjusted correctly")
     })
+    it("checks that withdraw blockers between solutions are reset", async () => {
```

Comment:
This is really testing an implementation detail. I think instead we should test that you can withdraw your funds even if the token was traded on your behalf in the first solution of the test (of it was undone in the second solution).